### ISSUE TITLE
`medsam_train_native.py`: `grad_norm` stuck at 0

### ISSUE THREAD

**Issue by turtle2-blip** [jp4474/HematomaSegmentation-VolumePrediction#2]
![image](https://github.com/jp4474/HematomaSegmentation-VolumePrediction/assets/119133740/543e5967-ea99-4e38-9e2e-3c664f90fe4d)

**Comment by jp4474** [comment]
fixed. use the new script. may need to play around with learning rate for optimal results.

{'val_loss': 0.006134545441628212, 'val_dice_score': 0.8366447176252093, 'val_jaccard_score': 0.7468266912869045}
{'loss': 0.0063, 'grad_norm': 0.010984303429722786, 'learning_rate': 0.00046, 'epoch': 0.8}
.
.
.
{'val_loss': 0.005807923175217141, 'val_dice_score': 0.8571935154142833, 'val_jaccard_score': 0.7660514314969381}
{'loss': 0.0057, 'grad_norm': 0.008016272448003292, 'learning_rate': 0.0002, 'epoch': 6.0}

loss and the metrics are improving.
